.github/workflows/test.yml

```diff
@@ -34,7 +34,11 @@ jobs:
     -
       name: Run editorconfig-checker
       run: editorconfig-checker
-
+    -
+      name: Run black python code formatter
```

Comment:
This action is just doing checks, right? It will flag any issues and present them to the PRing user? (note to self - install black locally) Can we change this text to something like `Check python code formatting with black` ?

https://black.readthedocs.io/en/stable/integrations/github_actions.html Suggests default options passed are `--check --diff`

Reply:
Yes. The current config is using `--check --diff`:
![image](https://user-images.githubusercontent.com/1385443/192111975-2230d779-c83f-4824-afc0-2c80bf15590c.png)

Note: _if you want to change the options, I suggest using `--check --diff --color`._

Reply:
Ha. I didn't even think to look at the last check run 🙈

```suggestion
      name: Check python code formatting with black
```